common/src/java/com/zimbra/common/account/ZAttrProvisioning.java

```diff
@@ -1664,6 +1664,24 @@ public static PrefCalendarInitialView fromString(String s) throws ServiceExcepti
         public boolean isYear() { return this == year;}
     }
 
+    public static enum PrefCalenderScaling {
+        _0("10"),
+        _5("15"),
+        _10("30");
+        private String mValue;
+        private PrefCalenderScaling(String value) { mValue = value; }
+        public String toString() { return mValue; }
+        public static PrefCalenderScaling fromString(String s) throws ServiceException {
+            for (PrefCalenderScaling value : values()) {
+                if (value.mValue.equals(s)) return value;
+             }
+             throw ServiceException.INVALID_REQUEST("invalid value: "+s+", valid values: "+ Arrays.asList(values()), null);
```

Comment:
add space before & after string concatenation or use String.format

Reply:
It's an auto-generated code.